neqo-http3/src/push_controller.rs

```diff
@@ -475,6 +475,10 @@ impl RecvMessageEvents for RecvPushEvents {
         );
     }
 
+    fn priority_update(&self, stream_id: u64, priority: Priority) {
+        self.push_handler.borrow_mut().new_stream_event(self.push_id, Http3ClientEvent::PriorityUpdate { stream_id, priority});
+    }
+
```

Comment:
why you need this. client cannot receive priority updates for push streams.

Reply:
I've added it so that the client can terminate the connection instead of ignoring the frame. From [Section 7.2](https://httpwg.org/http-extensions/draft-ietf-httpbis-priority.html#section-7.2)

> PRIORITY_UPDATE frames of either type are only sent by clients. If a client receives a PRIORITY_UPDATE frame, this MUST be treated as a connection error of type H3_FRAME_UNEXPECTED.